root/types.js

```diff
@@ -19,7 +19,7 @@ type CatalystContextT = {
 
 type CatalystStashT = {
   instruments_by_type?: {
-    [number]: Array<InstrumentT>,
+    [number]: ?Array<InstrumentT>,
     unknown: Array<InstrumentT>,
```

Comment:
The above change should apply to `unknown` as well, to cover the idyllic case where the type of every instrument is known.